### PR TITLE
Small clarification for default's service.name for exporters.

### LINKED ISSUE #1237: Jaeger exporter service.name clarification needed
**What are you trying to achieve?**

#1222 includes language that requires the Jaeger exporter to use the `service.name` attribute of the Resource, but does not explain how the exporter should behave if that attribute is missing from the Resource.

Should the exporter be configured with a fallback service name, or should it fail to export any spans without a valid Resource `service.name` entry?

### LINKED ISSUE #1380: Zipkin spec should suggest to use Resource.GetDefault
These 2 PRs modified default resource requirements. 
https://github.com/open-telemetry/opentelemetry-specification/pull/1269
https://github.com/open-telemetry/opentelemetry-specification/pull/1294

Zipkin exporter spec must be updated to reflect this change.
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk_exporters/zipkin.md#service-name

### PR DESCRIPTION
**Pull request by carlosalberto**
Fixes #1237 
Fixes #1380 

Small clarification on using the default `Resource` is no `service.name` was used.

PS - Don't think we need a CHANGELOG entry for this, but happy to add one if needed.

### CODE REVIEW

**Review by jkwatson — APPROVED** [review]

File: specification/trace/sdk_exporters/jaeger.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ single process and exporters need to handle this case accordingly.
 Critically, Jaeger backend depends on `Span.Process.ServiceName` to identify the service
 that produced the spans. That field MUST be populated from the `service.name` attribute
 of the [`service` resource](../../resource/semantic_conventions/README.md#service).
+If no `service.name` was specified, that field MUST be populated from the
```
Comment (Oberon00) [inline specification/trace/sdk_exporters/jaeger.md:46]:
```suggestion
If no `service.name` is contained in a Span's Resource, that field MUST be populated from the
```

**Review by Oberon00 — COMMENTED** [review]
Nit: "was specified" sounds a bit like there may be another way beyond resources to specify a service name.

**Review by Oberon00 — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

File: specification/trace/sdk_exporters/zipkin.md [hunk 2]
```diff
@@ -53,7 +53,9 @@ and Zipkin.
 
 Zipkin service name MUST be set to the value of the
 [resource attribute](../../resource/semantic_conventions/README.md):
-`service.name`. In Zipkin it is important that the service name is consistent
+`service.name`. If no `service.name` is contained in a Span's Resource, it MUST be populated from the
+[default](../../resource/sdk.md#sdk-provided-resource-attributes) `Resource`.
+In Zipkin it is important that the service name is consistent
```
Comment (cijothomas) [inline specification/trace/sdk_exporters/zipkin.md:58]: few lines down, it says "Zipkin MUST allow to configure the default "fall back" name". We need to remove that as well, as the only source from which service name should be obtained is the Resource.

Comment (carlosalberto) [reply specification/trace/sdk_exporters/zipkin.md:58]: Ooh, good catch! Will adjust that part, thanks! Updated.

**Review by cijothomas — APPROVED** [review]

**Review by andrewhsu — APPROVED** [review]
LGTM

**Comment by cijothomas** [comment]
> PS - Don't think we need a CHANGELOG entry for this, but happy to add one if needed

I'd add it, as this is easy to overlook by language SIGs. A changelog would make it more easy.

File: CHANGELOG.md [hunk 3]
```diff
@@ -52,6 +52,9 @@ Updates:
   ([#1339](https://github.com/open-telemetry/opentelemetry-specification/pull/1339))
 - Trace Exporters: Fix TODOs in Jaeger exporter spec
   ([#1374](https://github.com/open-telemetry/opentelemetry-specification/pull/1374))
+- Clarify that Jaeger/Zipkin exporters must to rely on the default Resource to
```
Comment (iNikem) [inline CHANGELOG.md:55]:
```suggestion
- Clarify that Jaeger/Zipkin exporters must rely on the default Resource to
```

**Pull request merged**